docs/docs/examples/llm/anthropic.ipynb

```diff
@@ -145,6 +145,83 @@
     "print(resp)"
```

Comment:
<p>nit: I think we should mention here that supplying <code>region</code> and <code>project_id</code> would result in using a <code>VertexAI</code> client for Anthropic LLMs, in case its not immediately obvious to our users.    </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/run-llama/llama_index/pull/15661/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
<p>Agreed, will add that</p>